CHANGELOG.md

```diff
@@ -385,6 +385,16 @@ more discussion).
   now you can still use them by prefixing `Ferrite.`, e.g. `Ferrite.getweights`.)
   ([#754][github-754])
 
+- The `onboundary` function have been removed ([#924][github-924]). Instead of first
```

Comment:
```suggestion
- The `onboundary` function and the associated `boundary_matrix` in the `Grid` datastructure have been removed ([#924][github-924]). Instead of first
```